_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,986 @@
+  [offers protocol][topic offers].
+
+- [Core Lightning 23.11][nov cln] provided additional flexibility to the
+  _rune_ authentication mechanism, improved backup verification, new and
+  features for plugins.
+
+- [Bitcoin Core 26.0][dec bcc] included experimental support for the
+  [version 2 transport protocol][topic v2 p2p transport], support for
+  [taproot][topic taproot] in [miniscript][topic miniscript], new RPCs
+  for working with states for [assumeUTXO][topic assumeutxo], and an
+  experimental RPC for processing [packages of transactions][topic
+  package relay].
+
+</div>
+
+## October
+
+{:#pss}
+Gijs van Dam posted [research results and code][oct pss] about _payment
+splitting and switching_ (PSS). His code allows nodes to split incoming
+payments into [multiple parts][topic multipath payments], which can take
+different routes before reaching the final recipient. For instance, a
+payment from Alice to Bob could be partly routed through Carol. This
+technique significantly hinders balance discovery attacks, where
+attackers probe channel balances to track payments across the network.
+Van Dam's research showed a 62% decrease in information gained by
+attackers using PSS.  Additionally, PSS offers increased Lightning
+Network throughput and may help mitigate [channel jamming attacks][topic
+channel jamming attacks].
+
+{:#sidepools}
+Developer ZmnSCPxj [proposed][oct sidepool] a concept called _sidepools_
+that aims to enhance LN's liquidity management. Sidepools involve
+multiple forwarding nodes depositing funds into a multiparty offchain
+state contract similar to LN channels. This allows funds to be
+redistributed among participants offchain. For instance, if Alice, Bob,
+and Carol each start with 1 BTC, the state can be updated so that Alice
+has 2 BTC, Bob 0 BTC, and Carol 1 BTC.  Participants would still use and
+advertise regular LN channels, and if those channels became imbalanced,
+a rebalancing can be done through an offchain peerswap within the state
+contract. This method is private to participants, requires less onchain
+space, and likely eliminates offchain rebalancing fees, thus improving
+revenue potential for forwarding nodes and reliability for LN payments.
+However, it requires a multiparty state contract, which is untested in
+production.  ZmnSCPxj suggests using [LN-Symmetry][topic eltoo] or
+duplex payment channels as a foundation, with duplex channels being more
+feasible despite their inability to remain open indefinitely.
+
+{:#assumeutxo}
+October saw the [completion][oct assumeutxo] of the first phase of the
+[assumeUTXO project][topic assumeutxo], containing all the remaining
+changes necessary to both use an assumedvalid snapshot chainstate and do
+a full validation sync in the background. It makes UTXO snapshots
+loadable via an RPC.  Although the feature set will not be available on
+mainnet until activated, this merge marked the culmination of a
```

Comment:
"Activated" makes it sound like it were a softfork, rather than it being unavailable due to no mainnet UTXO set snapshot being shipped with the latest release